packages/tangle/tangle.go

```diff
@@ -95,6 +97,7 @@ func (t *Tangle) Configure(options ...Option) {
 			Store:                        mapdb.NewMapDB(),
 			Identity:                     identity.GenerateLocalIdentity(),
 			IncreaseMarkersIndexCallback: increaseMarkersIndexCallbackStrategy,
+			CacheTimeProvider:            database.NewCacheTimeProvider(-1),
```

Comment:
Yeah, I also thought about that, here is what stopped me:
1. I think that the default behavior should be the same as what we have now (cache is enabled)
2. It would be weird that the tangle has default cache time 0, but the GoShimmer node itself has cache time enabled (-1) in case no one passed a CLI arg or config file.
3. The fact that we initialize the tangle in a single place won't stay forever, I would like to refactor our plugin system and get rid of singletons in the future. So an inner-level code like the tangle package shouldn't make assumptions that It gets initialized from a single place. 
4. In case a developer forgot to pass the option it's better to run unit tests with cache than to run production code without cache.

Reply:
However, it is also confusing for people who read only production code to see that the default is 0 and later gets overridden... people who see this won't know it is for tests..

I would even prefer to not have a default for this, and test writers who forget about the CacheTimeProvider will be reminded by an exception... But you can argue that this is also ugly...

I think the better solution would be to always use a testutil to initialize the tangle and the ideal solution is to mock the tangle in unit tests that just need it as a dependency (do we want to do this)?... But this is for another PR...

So as for right now, I would vote to just remove the default option because I think code is read more often than it is written.
I just think it is better to force test writers to think about using `newTestTangle` or mocks, rather to just have them incorrectly initialize the tangle in tests

Reply:
I am okay with both options: remove the option completely or keep it with cache enabled (-1). @GalRogozinski up to you!

Reply:
I'd favor @GalRogozinski's suggestion to leave it away. So it is a coding error when you forget to specify it and it can't be forgotten in unit tests :)